lib/sgpres.gi

```diff
@@ -2869,10 +2867,12 @@ local ded,offset,pair,alpha,x,p,w;
         fi;
       od;
     fi;
+    # separate 'if' check, as the `break;` only ends innermost loop
     if p[alpha]=alpha then
       alpha:=DATA.ct[x+offset][alpha]; # beta
       if p[alpha]=alpha then
-        for w in DATA.ccr[x+offset] do
+        # AH, 9/13/18: It's R^c_{x^-1}, so -x
+        for w in DATA.ccr[offset-x] do
```

Comment:
Question: I assume this is what the commit message calls "Also fixed wrong index in tracing backwards." You also wrote that this PR is purely about performance, not fixing a bug. So, I gather this means that the above bug could not have lead to incorrect results?

Reply:
This is code to trace relators through the coset table. The wrong sign used relators starting with x, not with x^-1 (Holt Handbook, p.166, code line 11), which will not give a new result here. As consequence, the coset enumeration will take much longer to terminate (potentially might not terminate), but as it is tracing of valid relators this cannot cause wrong results.

Reply:
Thanks for the explanation